conf/license.php

```diff
@@ -14,23 +14,23 @@
     'url'  => 'http://creativecommons.org/licenses/publicdomain/',
 );
 $license['cc-by'] = array(
-    'name' => 'CC Attribution 3.0 Unported',
-    'url'  => 'http://creativecommons.org/licenses/by/3.0/',
+    'name' => 'CC Attribution 4.0 International',
```

Comment:
[Apparently not](https://wiki.creativecommons.org/wiki/Frequently_Asked_Questions#What_are_the_international_.28.E2.80.9Cunported.E2.80.9D.29_Creative_Commons_licenses.2C_and_why_does_CC_offer_.E2.80.9Cported.E2.80.9D_licenses.3F)